tlsutil/tlsutil.go

```diff
@@ -57,7 +57,6 @@ func WithProfile(p Profile) Option {
 func WithCertificates(certs []tls.Certificate) Option {
 	return func(config *tls.Config) {
 		config.Certificates = append(config.Certificates, certs...)
-		config.BuildNameToCertificate()
```

Comment:
Looks like we're removing because this func was deprecated, and we don't need a replacement?

Reply:
That's my theory! To be honest, we don't use `tlsutil` in launcher, so I didn't angst much about it